api/src/components/coauthor/controller.ts

```diff
@@ -1,12 +1,23 @@
 import * as coAuthorService from 'coauthor/service';
-import * as email from 'email';
 import * as I from 'interface';
+import * as email from 'email';
 import * as response from 'lib/response';
 import * as publicationService from 'publication/service';
 
-export const create = async (
-    event: I.AuthenticatedAPIRequest<I.CreateCoAuthorRequestBody, undefined, I.CreateCoAuthorPathParams>
-): Promise<I.JSONResponse> => {
+
+export const get = async (event: I.AuthenticatedAPIRequest<undefined, undefined, I.CreateCoAuthorPathParams>) => {
```

Comment:
although admittedly its not consistently applied yet across the project, it would be nice to get into the habit of adding return values where we are editing a particular file.